STRAPI_UPDATE_STRATEGY.md

```diff
@@ -0,0 +1,147 @@
+# Strapi Update Strategy
+
+This document outlines the process for updating Strapi from the current version (v4.25.8) to newer versions.
+
+## Pre-Update Checklist
+
+### 1. Review Official Documentation
+
+- **Critical**: Check the [Strapi Migration Guide](https://docs-v4.strapi.io/dev-docs/migration-guides) for breaking changes
+- Review the [Strapi Changelog](https://github.com/strapi/strapi/releases) for all versions between current and target version
+- Pay special attention to any database migration requirements
+
+### 2. Plugin Version Alignment
+
+- Ensure all standard Strapi plugins match the target Strapi version
+- Update plugin versions in `package.json` to align with the new Strapi version
+- Check compatibility of custom plugins with the new version
+
+## Update Process
+
+### 3. Homepage Customization (patch-package)
+
+We currently override the Strapi homepage using patch-package. After each update:
+
+1. **Locate Homepage Files**: Navigate to `node_modules/@strapi/admin/dist/chunks/`
+2. **Find Homepage Chunks**: Look for homepage-related files (may be multiple files for different locales)
+   - File names may change between versions
+   - Search for files containing homepage components
+3. **Apply Customizations**: Modify the homepage files as needed
+4. **Update Patch File**:
+
+   ```bash
+   # Remove old patch file
+   rm patches/@strapi+admin+*.patch
+
+   # Generate new patch file
+   npx patch-package @strapi/admin
+   ```
+
+### 4. React Version Compatibility
+
+If Strapi requires React updates:
+
+- Test the Strapi dashboard thoroughly
+- Pay special attention to non-standard Strapi plugins
+- Verify compatibility with custom-developed plugins
+
+## Testing Strategy
+
+### Local Environment Testing
+
+#### Non-Docker Testing
+
+Test development and build processes without Docker:
+
+```bash
+# Interactive development setup
+yarn development
+# Select: Dev → Choose app/project → Test functionality
+
+# Interactive build setup
+yarn development
+# Select: Build → Choose app/project → Verify build success
+```
+
+#### Docker Environment Testing
+
+Test both development and production Docker environments:
+
+```bash
+# Interactive Docker setup
+yarn development
+# Select: Docker → Choose project → Choose environment (dev/prod) → Choose action (up --build)
+```
```

Comment:
Dit stuk zou ik eruit laten; het is nogal obvious, en dan heb je minder risico dat deze documentatie out of sync gaat lopen